packages/shared/src/domain/rate-limiting/index.ts

```diff
@@ -1,22 +1,23 @@
 import { z } from "zod";
+import dayjs from "dayjs";
+import { MetriportError } from "../../error/metriport-error";
 
-export const oneMinuteInMs = 60000;
+export const globalWindow = dayjs.duration(60000, "milliseconds");
```

Comment:
Can we use `dayjs.duration(1, "minute");`? 😃

Reply:
Done.